website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -0,0 +1,261 @@
+---
```

Comment:
This doc is very important at the moment because we have just made the change. However, I think a better place for this long term is the reference section